tests/cross/src/test/scala/tests/pc/CompletionOverrideAllSuite.scala

```diff
@@ -0,0 +1,366 @@
+package tests.pc
+
+import tests.BaseCompletionSuite
+
+object CompletionOverrideAllSuite extends BaseCompletionSuite {
+
+  override def beforeAll(): Unit = {
+    indexJDK()
+  }
+
+  check(
+    "simple",
+    """|package example
+       |
+       |trait Foo {
+       |  def foo: Int
+       |  def bar: Int
+       |}
+       |object Main {
+       |  val x = new Foo {
+       |    def@@
+       |  }
+       |}
+       |""".stripMargin,
+    """|Implement all members (2 total)
+       |""".stripMargin,
+    filter = _.contains("Implement")
+  )
+
+  // Note: ckipp01 the indentation for the completions on the edits
+  // look a bit odd, but it seems the tests aren't respecting the actual
+  // indentation as these all work locally as expected, and when the actual
+  // indent is added, the necessary indent is unnecessarily doubled
```

Comment:
If the test behavior is different than the actual one, this may be a bug in 
https://github.com/scalameta/metals/blob/f7dc2c5d6732bb484420451cccc8e386807d49fd/tests/mtest/src/main/scala/tests/TextEdits.scala#L13-L34

Reply:
I'll take a closer look at this here to see if I can find anything to explain what's happening. Thanks!